terraform/node_resource_plan_orphan.go

```diff
@@ -48,6 +49,14 @@ func (n *NodePlannableResourceInstanceOrphan) Execute(ctx EvalContext, op walkOp
 	}
 }
 
+func (n *NodePlannableResourceInstanceOrphan) ProvidedBy() (addr addrs.ProviderConfig, exact bool) {
+	if n.Addr.Resource.Resource.Mode == addrs.DataResourceMode {
+		// indicate that this node does note require a configured provider
```

Comment:
```suggestion
		// indicate that this node does not require a configured provider
```

😛 
Change looks good, though; it's weird but weird in a direction I'm not surprised to see us go.